identityClient/api.go

```diff
@@ -893,3 +893,28 @@ type RealmSettingsUpdateRequest struct {
 	MFAAvailable        *[]string `json:"mfa_available,omitempty"`
 	EmailLookupsEnabled *bool     `json:"email_lookups_enabled,omitempty"`
 }
+
+// CreateAccessRequestRequest wraps parameters for creating a new open Access Request
+type CreateAccessRequestRequest struct {
+	Groups         []AccessRequestGroup `json:"groups"`
+	Reason         string               `json:"reason"`
+	RealmName      string               `json:"realm_name"`
+	AccessDuration int                  `json:"ttl"`
```

Comment:
can the JSON be `access_duration`

Reply:
yeah but then I got to change the api spec and at least there it has a description of what the field is for 😅

Reply:
Not worth 3 PRs